### DISCUSSION TITLE
Add digital products

### DISCUSSION THREAD

**Discussion by jonassiewertsen** [jonassiewertsen/statamic-butik#165] [Ideas]
Make it possible to sell digital products, so they can be downloaded. 

**What kind of features would you like to see with digital products?**

**Comment by ghost** [comment]
Digital products need a UUID which can serve as a license code if for their specific sales application.

Digital product needs a downloadable file(s) input - link to external file (stored on aws bucket or somewhere)

Digital products need a download page which calls the file, not a direct link to file (security reasons)

**Reply by ghost** [reply]
Also, digital products could benefit from having a subscription module
